docs/dev_guide/migrate-a-project/generate-compilation-db.rst

```diff
@@ -115,6 +115,20 @@ in the build directory:
 
    CMake 3.10 or higher is recommended as it provides better support for CUDA source code.
 
+Generate a Compilation Database for Setuptools-Based Projects
+--------------------------------------------------------
+Setuptools is a package development library in Python for facilitating packaging, distribution and installation of Python projects. For projects with setuptools as their build system, the steps to generate a compilation database are as follows:
+
+#. Generate the build log for the project using the verbose flag.

```

Comment:
Please be more detail on how to apply the "verbose" flag,  Give example ok is OK.